test/e2e/components/vl-multi-select.js

```diff
@@ -1,6 +1,140 @@
-const { VlElement } = require('vl-ui-core').Test;
+const { VlSelect } = require('vl-ui-select').Test;
+const { By, Key } = require('selenium-webdriver');
```

Comment:
"By" kan ook uit vl-ui-core.Test.Setup gehaald worden, Key weet ik even niet vanbuiten. Dan gebruik je een rechtstreekse dependency ipv. een transitive dependency.

Reply:
`By` is aangepast. `Key` werkt niet.

Reply:
Ah ok. Stom dat we een stuk van selenium wel exposen en een stuk dan weer niet. 
Misschien moeten we dat ook eens rechttrekken, @coemans .

Reply:
Inderdaad @SanderKleykens al dacht ik dat we met de huidige exports zouden toekomen. Geen probleem om alles te exposen of exports toe te voegen.